src/core/utils/createApolloClient.ts

```diff
@@ -39,6 +43,43 @@ export default (host: string): ApolloClient<unknown> => {
                 clearToken();
                 window.location.reload();
             }
+        }
+
+        if (networkError) {
+            console.log(`[Network error]: ${networkError}`);
```

Comment:
should this be a `console.error` to make it stand out when debuging? (I know the Apollo docs say console.log but this seems like something we might want to catch with newrelic)?

Reply:
Ditto for line 35

Reply:
I believe this was existing code prior to the PR. While it's a valid concern it should probably become it's own ticket as it's out of scope of this PR

Reply:
https://github.com/libero/reviewer/issues/674